src/main/java/edu/ucdavis/dss/ipa/repositories/SectionGroupCostInstructorRepository.java

```diff
@@ -41,4 +43,9 @@
     List<SectionGroupCostInstructor> findbyWorkgroupIdAndYear(@Param("workgroupId") long workgroupId, @Param("year") long year);
 
     SectionGroupCostInstructor findByInstructorIdAndSectionGroupCostIdAndTeachingAssignmentId(Long instructorId, Long SectionGroupCostId, Long teachingAssignmentId);
-}
\ No newline at end of file
+
+    @Transactional
+    @Modifying
+    @Query("delete from SectionGroupCostInstructor sgci where sgci.id in ?1")
```

Comment:
Minor thing, I believe this could be "= ?1" instead of "in ?1" since it is only ever one id?